nameable.rb

```diff
@@ -0,0 +1,37 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Method not implemented'
+  end
+end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
+end
```

Comment:
- [ ] Please consider fixing this method, it returns the name in UPPER CASE, but the requirements state that it should be capitalized (First letter uppercase). 🤓 🔧 

![image](https://github.com/addisalemseifu/library-app-ruby/assets/77069011/a74d9436-f2ff-4937-87c9-53365e2aed3d)

The expected `correct_name` in this case is: `Maximilianus`.